modules/ROOT/attachments/specs/enotices2-ted-europa-eu-esenders.yaml

```diff
@@ -146,7 +159,12 @@ paths:
             Possible issues:
             - Missing Authorization header in request
             - Notice with the same noticeId and versionId already exists.
+            - NoticeAuthorEmail and noticeAuthorLocale are mandatory.
+            - Notice author email is not valid.
+            - Notice parameter cannot be empty.
```

Comment:
The items in this list sometimes indicate what is required ("... are mandatory"), and sometimes what is wrong (... cannot be empty). This makes it confusing.
We should consistenly use one formulation. I think indicating what is wrong is clearer.

Reply:
OK